src/frontend/app/store/actions/route.actions.ts

```diff
@@ -62,9 +62,9 @@ export class DeleteRoute extends BaseRouteAction {
   constructor(
     public guid: string,
     public endpointGuid: string,
+    appGuid?: string,
```

Comment:
app guid now shouldn't be optional, as it's needed by the app summary reducer. It's not supplied in the space routes list delete action.